guide.rst

```diff
@@ -259,41 +259,20 @@ without a selector and referred to from other messages, like the ``about``
 message above.
 
 
-Storing Additional Information
-==============================
-
-::
-
-    brand-name = Firefox
-        [gender] masculine
-
-    opened-new-window = { brand-name[gender] ->
-       *[masculine] { brand-name } otworzyl nowe okno.
-        [feminine] { brand-name } otworzyla nowe okno.
-    }
-
-Traits can be used to describe parameters of the entity that can be then
-used in other selectors.
-
-Imagine an entity ``brand-name`` that can be either *Firefox* or *Aurora*.  The 
-former is *masculine*, while the latter is *feminine*, so sentences that refer 
-to this entity may want to branch depending on the gender of it.
-
-
-HTML/XUL Attributes
-===================
+HTML Attributes
+===============
 
 ::
 
     login-input = Predefined value
-        [html/placeholder] example@email.com
-        [html/aria-label]  Login input value
-        [html/title]       Type your login email
+        .placeholder = example@email.com
+        .aria-label = Login input value
+        .title = Type your login email
```

Comment:
I remember talking to @flodolo about this: aligning values often times ends up creating diff noise and his recommendation was not to do it. And it will also make it easier to write a syntax-formatting tool for FTL that way :)